tests/conftest.py

```diff
@@ -105,13 +105,13 @@ def compileSolidity(relativeFilePath):
                 }
             },
             'settings': {
-                'remappings': [ 'ROOT=/augur-core/src' ]
+                'remappings': [ 'ROOT=%' % resolveRelativePath("/augur-core/src") ]
```

Comment:
I'm also quite surprised this actually works.  Shouldn't it be `resolveRelativePath('../src')`?